src/components/header.js

```diff
@@ -39,40 +39,72 @@ function Header({ slug }) {
   const [isOpen2, setIsOpen2] = useState(false);
 
   /* first dropdown items */
-  const navDropdownLinks1 = ['link-one', 'link-two']
+  const navDropdownLinks1 = ["link-one", "link-two"];
```

Comment:
Why are the paths hard coded here? I think they should be in the translation files

Reply:
There will be duplication if put in the translation files because regardless of what language they're in they will be directed to the same page.

Reply:
Okay. Maybe they should be in the constants folder?

Reply:
Excellent thought and reasoning. Links added to `/src/constants/links.js`.